pocketlint/tests/test_restructuredtext.py

```diff
@@ -343,6 +347,39 @@ def test_check_section_delimiter_bad_marker_length(self):
         self.assertEqual(expect, self.reporter.messages)
         self.assertEqual(1, self.reporter.call_count)
 
+    def test_semantic_newline_fullstop(self):
+        """Check for a sentence with a . and no new line."""
```

Comment:
this docstring does not help to much. 

see http://styleguide.chevah.com/testing.html#tests-description-docstrings

It should sound something like.

"""
When a full stop is not the last character from the line, it is considered a semantic newline violation and an error is reported.
"""